guides/admin/timescale-migration.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Migrating data Coder's database
+description: Learn how to migrate data from Coder's built-in PostgreSQL database
+to an external PostgreSQL instance.
+---
+
+By default, Coder deploys a built-in database in the installation's Kubernetes
+namespace. We recommend using this database _only_ for evaluation purposes.
+
+At the end of your evaluation period, you may need to migrate the data from the
+built-in database to an out-of-cluster PostgreSQL database for production use.
+This article will walk you through the process of doing so.
+
+> You must be a cluster admin for your Kubernetes cluster.
+
+## Migration Steps
+
+1. Access the database pod and dump the database into a file:
+
+   ```console
+   kubectl exec -it statefulset/timescale -n coder -- pg_dump -U coder -d coder > backup.sql
+   ```
+
+1. **Optional**: If your database is large, you can truncate Coder's telemetry,
+   metrics, and audit log data to reduce the file size:
+
+   ```psql
+   TRUNCATE metric_events;
+   TRUNCATE environment_stats;
+   TRUNCATE audit_logs;
+   ```
+
+1. Access your PostgreSQL instance and create user and database `coder`
```

Comment:
I'm a bit confused by this; they should create a `coder` user AND a database `coder`?

Reply:
yeah, PostgreSQL has user, database, and schema. Multiple users can have access to a database, and database can have multiple schemas. So yes, people should create a user name coder that has access to a database, also named coder